### PR TITLE
Refresh what-is-openstack page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Refresh /openstack/what-is-openstack page

## QA

- Check https://ubuntu-com-11547.demos.haus/openstack/what-is-openstack against [copy docs](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit#)
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5173

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/openstack/what-is-openstack.html [hunk 1]
```diff
@@ -379,171 +372,80 @@ <h2>Companies involved in OpenStack</h2>
...
+    </p>
+  </div>
+  <div class="row u-equal-height">
+    <article class="col-6 p-card">
+      <div class="p-card__title">
+        <img src="https://brandix.com/images/logo/logo-pt.jpg" />
       </div>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        In 2022, Pacific Textiles Holdings Limited - one of the world’s largest fabric mills - announced that it’s going to use Canonical’s <a href="/openstack">Charmed OpenStack</a> and <a href="/kubernetes">Charmed Kubernetes</a> to give its developers easy access to technologies like cloud-native apps, Kubernetes, AI/ML, and others alike.
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:394]:
```suggestion
        In 2022, Pacific Textiles Holdings Limited &ndash; one of the world’s largest fabric mills - announced that it’s going to use Canonical’s <a href="/openstack">Charmed OpenStack</a> and <a href="/kubernetes">Charmed Kubernetes</a> to give its developers easy access to technologies like cloud-native apps, Kubernetes, AI/ML, and others alike.
```

File: templates/openstack/what-is-openstack.html [hunk 2]
```diff
@@ -379,171 +372,80 @@ <h2>Companies involved in OpenStack</h2>
...
+        <a href="/blog/pacific-textiles-drives-digital-transformation-with-new-infrastructure-for-legacy-and-cloud-native-applications">Read the announcement&nbsp;&rsaquo;</a>
       </p>
+    </article>
+    <article class="col-6 p-card">
+      <div class="p-card__title">
+        <img src="https://assets.ubuntu.com/v1/dda24c61-Nayatel.png" width="153"/>
+      </div>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        In 2022, Nayatel - one of Pakistan’s leading telecom providers - announced that it completed the deployment of Pakistan’s first local public cloud. Aiming to meet the demand for a homegrown public cloud service, Nayatel partnered with Canonical to build an <a href="/openstack/features">enterprise-grade OpenStack platform</a>.
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:407]:
```suggestion
        In 2022, Nayatel - one of Pakistan’s leading telecom providers &ndash; announced that it completed the deployment of Pakistan’s first local public cloud. Aiming to meet the demand for a homegrown public cloud service, Nayatel partnered with Canonical to build an <a href="/openstack/features">enterprise-grade OpenStack platform</a>.
```

File: templates/openstack/what-is-openstack.html [hunk 3]
```diff
@@ -379,171 +372,80 @@ <h2>Companies involved in OpenStack</h2>
...
+    </article>
+  </div>
+  <div class="row u-equal-height">
+    <article class="col-6 p-card">
+      <div class="p-card__title">
+        <img src="https://assets.ubuntu.com/v1/2d2e1ab2-king-abdullah-university-of-science-kaust-logo.png" width="160" />
+      </div>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        n 2021, King Abdullah University of Science and Technology (KAUST) announced that it switched in away from its homegrown solution and adopt third-party support. Ubuntu familiarity alongside superior economics made Charmed OpenStack and <a href="/openstack/support">Ubuntu Advantage</a> the natural choice for KAUST.
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:422]:
```suggestion
        In 2021, King Abdullah University of Science and Technology (KAUST) announced that it switched in away from its homegrown solution and adopt third-party support. Ubuntu familiarity alongside superior economics made Charmed OpenStack and <a href="/openstack/support">Ubuntu Advantage</a> the natural choice for KAUST.
```

File: templates/openstack/what-is-openstack.html [hunk 4]
```diff
@@ -379,171 +372,80 @@ <h2>Companies involved in OpenStack</h2>
...
+      </p>
+      <br />
+      <p>
+        <a href="/engage/openstack-cloud-kaust">Read the case study</a>
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:426]:
```suggestion
        <a href="/engage/openstack-cloud-kaust">Read the case study&nbsp;&rsaquo;<</a>
```

File: templates/openstack/what-is-openstack.html [hunk 5]
```diff
@@ -669,9 +566,9 @@ <h2>Why Canonical’s Charmed OpenStack?</h2>
     <ul class="p-matrix is-trisected">
       <li class="p-matrix__item">
         <div class="p-matix__content">
-          <h5>Best price-performance guaranteed</h5>
+          <h5>Price-performance guaranteed</h5>
           <p>
-            Engineered for the best price-performance, Canonical’s Charmed
+            Engineered for price-performance, Canonical’s Charmed
             OpenStack delivers TCO reduction, while maximising the performance
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:572]:
```suggestion
            OpenStack delivers TCO reduction while maximising the performance
```

File: templates/openstack/what-is-openstack.html [hunk 6]
```diff
@@ -762,34 +659,13 @@ <h5>Up to ten years of security updates</h5>
...
-        width="1504",
-        height="1085",
-        hi_def=True,
-        loading="auto",
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
 <section class="p-strip--light">
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:662]:
```suggestion
<section class="p-strip">
```

**Review by minkyngkm — COMMENTED** [review]
Looking good @carkod! I made suggestions and here are a few minor things to be updated.

1.  Should this chart be removed? I can't this chart in the copy doc.
image
2. In line 589, a hyphen is missing
3. In the "OpenStack is big" section, some hyphens are missing as well.

**Comment by carkod** [comment]
> Looking good @carkod! I made suggestions and here are a few minor things to be updated.
> 
> 1. Should this chart be removed? I can't this chart in the copy doc.
> 
> image
> 
> 2. In line 589, a hyphen is missing 3. In the "OpenStack is big" section, some hyphens are missing as well.

Thanks, all comments addressed.
Removed.

File: templates/openstack/what-is-openstack.html [hunk 7]
```diff
@@ -343,207 +336,92 @@ <h2>Companies involved in OpenStack</h2>
...
+      <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms project</a>
+      and one of the biggest contributors to OpenStack all the time.
+    </p>
+  </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-8">
+<section class="p-strip">
```
Comment (minkyngkm) [inline templates/openstack/what-is-openstack.html:349]: Maybe this section to `p-strip--light` and the other sections below the other way around? 

```suggestion
<section class="p-strip--light">
```

**Review by minkyngkm — APPROVED** [review]
Thanks Carlos! There are two `open stack` that needs to be updated to `open-stack` in the "OpenStack is big" section! Other than that all LGTM! 👍

**Comment by carkod** [comment]
> Thanks Carlos! There are two `open stack` that needs to be updated to `open-stack` in the "OpenStack is big" section! Other than that all LGTM! +1

Done. Thanks

**Pull request merged**